src/Bitmovin/api/factories/manifest/HlsManifestFactory.php

```diff
@@ -304,20 +279,111 @@ private static function createPlaylistFileName($segmentPath, $hlsConfigurationFi
     }
 
     /**
-     * @param $configurations
-     * @return string
+     * @param JobContainer      $jobContainer
+     * @param AbstractHlsOutput $output
+     * @return array|null|string
      */
-    private static function getAudioGroupId($configurations)
+    private static function getAudioGroupIds($jobContainer, AbstractHlsOutput $output)
     {
-        $audioGroupId = null;
-        foreach ($configurations as $codecConfigContainer)
+        $configurations = array();
+        foreach ($jobContainer->encodingContainers as $container)
         {
-            if ($codecConfigContainer->apiCodecConfiguration instanceof AACAudioCodecConfiguration)
+            $configurations = array_merge($configurations, self::getConfigurationsForEncoding($container, $output));
+        }
+        if ($output->audioVideoGroups == null)
+        {
+            $audioGroupId = null;
+            foreach ($configurations as $codecConfigContainer)
             {
-                $audioGroupId = 'audio';
+                if ($codecConfigContainer->apiCodecConfiguration instanceof AACAudioCodecConfiguration)
+                {
+                    $audioGroupId = 'audio';
```

Comment:
How does this work if there are multiple qualities with different audio group ids?